src/components/Facets.tsx

```diff
@@ -0,0 +1,80 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
+import { FilterGroup, FilterGroupCssClasses } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link Facets}.
+ *
+ * @public
+ */
+export interface FacetsCssClasses extends FilterGroupCssClasses {
+  container?: string,
+  divider?: string
+}
+
+/**
+ * Props for the {@link Facets} component.
+ *
+ * @public
+ */
+export interface FacetsProps {
+  /** {@inheritDoc FilterGroupProps.collapsible} */
+  collapsible?: boolean,
+  /** {@inheritDoc FilterGroupProps.defaultExpanded} */
+  defaultExpanded?: boolean,
+  /** {@inheritDoc FilterGroupProps.searchable} */
+  searchable?: boolean,
+  /**
+   * Whether or not a search is automatically run when a filter is selected.
+   * Defaults to true.
+   */
+  searchOnChange?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: FacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays all facets applicable to the current vertical search.
+ *
+ * @param props - {@link FacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function Facets(props: FacetsProps) {
+  const { searchOnChange, customCssClasses = {}, ...filterGroupProps } = props;
+  const { container:containerCssClasses, divider, ...filterGroupCssClasses } = customCssClasses;
+  return (
+    <FacetsCompoundComponent searchOnChange={searchOnChange} className={containerCssClasses}>
+      {facets => {
+        const filteredFacets = facets.filter(f => f.options.length > 0);
+        return (<>
+          {filteredFacets.map((f, i) => {
+            const filterOptions = f.options.map(option => ({ ...option, label: option.displayName }));
```

Comment:
All this map does is copy `displayName` into `label` right? Can we just, use displayName instead?

Reply:
this is because CheckboxOption, use in FilterGroup, expects a label prop in its interface. StaticFilters also have label prop as agreed by product. Since StaticFilters and Facets both use FilterGroup, this mapping is needed in Facets for it to be consistent too with `label`

Reply:
Oh I see, did we want to wrap filterOptions in a useMemo then? Otherwise we're never getting reference equality since each time it renders it'll be a new array. Alternatively I don't mind if we accept `displayName` as an alias for `label` but I think Tom would be against that (cc: @tmeyer2115 )

Reply:
can't have useMemo inside callback. I will check in with product if we can use displayName instead in StaticFilters, I think switching `label` in Facets/StaticFilters/CheckboxOptionProps to `displayName` would be better align with the `Filter` interface we use in answers-headless and core, so we don't have to do additional mapping

Reply:
product is good with switching to displayName for all so I will update the pr!